README.md

```diff
@@ -10,69 +10,9 @@ To manage the Godot version for this project, we recommend using the [Godot Vers
 2. Initialize a new project using Godot 4.0.4 stable.
 3. Clone this repository into your project directory.
 
-## Branch Naming Convention
-
-Please follow this branch naming convention when creating branches for new features, bug fixes, or tasks:
-
-`{number of the ticket}-name-of-the-ticket`
```

Comment:
I think this explains it well maybe it would be something worth to keep

Reply:
Humm, I think, don't make sense to have in both side, I think we should have the documentation only on Wiki, to be simple to every one, for as dev, that only need to change in one place, and for other persons that only have to read in only one place.

What you think?

Reply:
yes what I meant was to include the ``{number of the ticket}-name-of-the-ticket`` in the wiki, but if you don't think its worth it its okay

Reply:
Ahhhhh ok you mean that, sorry